R/deploy-site.R

```diff
@@ -68,7 +67,6 @@ deploy_site_github <- function(
   tarball = Sys.getenv("PKG_TARBALL", ""),
   ssh_id = Sys.getenv("id_rsa", ""),
   repo_slug = Sys.getenv("TRAVIS_REPO_SLUG", ""),
-  host = "github.com",
```

Comment:
To be safe, maybe deprecate this argument?

Reply:
And can `repo_slug` be deprecated now too?

Reply:
I thought maybe since `host` was never in a CRAN release we could just remove it. I will definitely deprecate `repo_slug`.

Reply:
Oh I missed that; that makes sense.